lib/apogeeDetection/apogeeDetection.cpp

```diff
@@ -79,7 +79,7 @@ double ApogeeDetection::getAlt() {
 }
 
 bool ApogeeDetection::weAtMECOBro() {
-	if(acc_z - previousAcc_z <= -5)
```

Comment:
this is not robust enough. we want something that can handle random noise. use the same descending + outlook algorithm. but for acceleration data instead of altitude data